### DISCUSSION TITLE
go2rtc not starting streams on Wyze V3 cam with 4.36.9.139 firmware.

### DISCUSSION THREAD

**Discussion by ltomes** [gtxaspec/wz_mini_hacks#796] [Q&A]
Hi, I have installed the latest wz_mini_hacks software on one of my wyze cams.
The camera is running the proper 4.36.9.139 firmware (not a newer, incompatible firmware if I am understanding [the docs](https://github.com/gtxaspec/wz_mini_hacks/wiki/Firmware-Support#notes) correctly):
![image](https://github.com/user-attachments/assets/e4e47422-741a-433a-aa50-1dd32a4345f2)

Yet no streams are starting in go2rtc:
![image](https://github.com/user-attachments/assets/be4f9134-6e6a-4d53-a763-1231170ab24a)

I am (basically) running a stock configuration with the UI enabled. I am happy to try any configuration someone suggests. This is my first time trying to enable go2rtc on a wyze cam.

Here is my (redacted) configuration (as a txt file so it can upload), please let me know if you see any mistakes I made in configuration, I'm happy to try a stock configuration if that helps me debug the issue.
[wz_mini.conf.txt](https://github.com/user-attachments/files/17193660/wz_mini.conf.txt)

I see this error in the go2rtc logs:
```
9/30/2024, 3:22:29 PM | error | undefined error=streams: rtsp module disabled caller=/home/aag/Downloads/go2rtc/internal/mjpeg/init.go:166
-- | -- | --
```

**Comment by bjs-pdx** [comment]
Check go2rtc.yml:
```

# NOTE: USE /dev/video6 for Wyze Cam v2/PanV1!
  - exec:ffmpeg -fflags +genpts+flush_packets+nobuffer+bitexact -flags +low_delay -thread_queue_size 100 -analyzeduration 50 -probesize 50 -pix_fmt yuv420p -r 30 -i /dev/video1  -flags +global_header -movflags +empty_moov+default_base_moof+frag_keyframe -framerate 30 -video_size 1920x1080 -pix_fmt yuv420p -c:v copy -copyts -rtsp_transport tcp -f rtsp {output}
  - exec:ffmpeg -hide_banner -f alsa -ac 1 -i dsnooper0 -c:a copy -muxdelay 0 -muxpreload 0 -f rtsp {output}
  - exec:ffmpeg -hide_banner -f alsa -ac 1 -i dsnooper0 -c:a libfdk_aac -afterburner 1 -channels 1 -b:a 128k -profile:a aac_he -ar 16000 -strict experimental -muxdelay 0 -muxpreload 0 -ac:a 1 -f rtsp {output}
# libopus has higher cpu requirements, audio may studder.
#  - exec:ffmpeg -hide_banner -f alsa -ac 1 -i dsnooper0 -c:a libopus -muxdelay 0 -muxpreload 0 -ac:a 1 -af adelay=0|0 -f rtsp {output}
  360p:
# NOTE: USE /dev/video7 for Wyze Cam v2/PanV1!
  - exec:ffmpeg -fflags +genpts+flush_packets+nobuffer+bitexact -flags +low_delay -thread_queue_size 100 -analyzeduration 50 -probesize 50 -pix_fmt yuv420p -r 30 -i /dev/video2 -framerate 30 -video_size 640x360 -pix_fmt yuv420p -c:v copy -copyts -rtsp_transport tcp -f rtsp {output}
  - exec:ffmpeg -hide_banner -f alsa -ac 1 -i dsnooper2 -c:a copy -muxdelay 0 -muxpreload 0 -ac:a 1 -f rtsp {output}
  - exec:ffmpeg -hide_banner -f alsa -ac 1 -i dsnooper2 -c:a libfdk_aac -afterburner 1 -channels 1 -b:a 128k -profile:a aac_he -ar 16000 -strict experimental -muxdelay 0 -muxpreload 0 -ac:a 1 -f rtsp {output}
```

**Reply by ltomes** [reply]
I don't understand which settings are proper for a wyze v3 cam based on what I see in this file but since I haven't configured anything in this file specifically for the v3 I bet this is the issue. I'll see if I can find a config that works online.
